metacat-connector-cassandra/src/main/java/com/netflix/metacat/connector/cassandra/CassandraConnectorDatabaseService.java

```diff
@@ -85,8 +96,13 @@ public void create(
     public void delete(@Nonnull @NonNull final ConnectorContext context, @Nonnull @NonNull final QualifiedName name) {
         final String keyspace = name.getDatabaseName();
         log.debug("Attempting to drop Cassandra keyspace {} for request {}", keyspace, context);
-        this.executeQuery("DROP KEYSPACE IF EXISTS " + keyspace + ";");
-        log.debug("Successfully dropped {} keyspace", keyspace);
+        try {
+            this.executeQuery("DROP KEYSPACE IF EXISTS " + keyspace + ";");
+            log.debug("Successfully dropped {} keyspace", keyspace);
```

Comment:
Yes. A no-op which results in the users desired end state anyway. That being that the Database (keyspace) doesn't exist. It's cleaner than throwing an exception just to say what they wanted couldn't happen cause they're already in their desired state.